src/ReactiveUI/ReactiveUI.csproj

```diff
@@ -22,7 +22,7 @@
   <ItemGroup>
     <Compile Remove="Platforms\**\*.cs" />
     <None Include="Platforms\**\*.cs" />
-    <PackageReference Include="Splat" Version="12.*" />
+    <PackageReference Include="Splat" Version="11.*" />
```

Comment:
again v12?

Reply:
Absolutely the final version will have V12